extensions/mssql/src/sqlProjects/sqlProjectsService.ts

```diff
@@ -475,11 +475,11 @@ export class SqlProjectsService extends BaseService implements mssql.ISqlProject
 	/**
 	 * Move a folder and its contents within a project
 	 * @param projectUri Absolute path of the project, including .sqlproj
-	 * @param destinationPath Path of the folder, typically relative to the .sqlproj file
-	 * @param path Path of the folder, typically relative to the .sqlproj file
+	 * @param sourcePath Source path of the folder, typically relative to the .sqlproj file
+	 * @param destinationPath Destination path of the folder, typically relative to the .sqlproj file
 	 */
-	public async moveFolder(projectUri: string, destinationPath: string, path: string): Promise<azdata.ResultStatus> {
-		const params: contracts.MoveFolderParams = { projectUri: projectUri, destinationPath: destinationPath, path: path };
+	public async moveFolder(projectUri: string, sourcePath: string, destinationPath: string): Promise<azdata.ResultStatus> {
```

Comment:
Make sure all invocations to `moveFolder` have their source and destination path arguments in the right order since they were swapped here.

Reply:
There aren't any invocations yet :) Slowly adding everything needed into this PR.